File: dataset/human/c/file_4480.c
#include<stdio.h>
int main(void){
	int n,d_1,d_2;
	int cnt=0;
	scanf("%d",&n);
	while(n--){
		scanf("%d%d",&d_1,&d_2);
		if(cnt){
			if(d_1==d_2){
				cnt++;
			}
			else{
				cnt=0;
			}
		}
		else{
			cnt+=d_1==d_2;
		}
		if(3<=cnt){
			break;
		}
	}
	printf(3<=cnt?"Yes":"No");
	return 0;
}

File: dataset/human/python/file_4480.py
def f(i,n):
  ans =0
  cnt = i
  while n%cnt == 0:
    n//=cnt
    cnt*=i
    ans +=1
  while n%i ==0:
    n//=i
  return n,ans

n = int(input())
i = 2
ans = 0
while i <= int(n**0.5)+1:
  if n%i ==0:
    n,a = f(i,n)
    ans += a
  i += 1
  
if n >1:
  ans += 1
print(ans)
